### DISCUSSION TITLE
Is it possible to reflash this SNES repro?

### DISCUSSION THREAD

**Discussion by blueoasis** [sanni/cartreader#466] [Need help]
I'm on 9.4 and am trying to reflash a SNES repro cart in my library, so I decided to try to write to that, but when I do, I get the UNKNOWN FLASHROM error regardless of which option I pick under Flash Repro. I saw the information about soldering two points on the board for some Chinese repros, but my board seems totally different. Here are a couple of pics of the board. I wasn't able to find any information on it
![20220814_131151](https://user-images.githubusercontent.com/1864191/184551855-feb3c6d0-2061-44a9-8597-49c172b61c2d.jpg)
![20220814_131203](https://user-images.githubusercontent.com/1864191/184551857-36d4e90a-3815-4074-a5b3-93c9a7319a80.jpg)

**Comment by sanni** [comment]
MSP55LV160A is not supported. It could however just be a rebranded MX29LV160. To find out more you have to use a multimeter to check where each pin of the MSP55LV160A chip is connected to on the cartridge edge and then compare it to the MX29LV160 pinout from its datasheet.

**Reply by blueoasis** [reply]
Thanks for the response!

**Reply by Chipchip256** [reply]
Tenho 5 dessas placas e estou tentando gravar, so que nao existe datasheet do MSP55LV160, e buscando os pinout descobri que o que se aproxima da mesma pinagem é o MBM29LV800BA tenho que estudar o datasheet pra saber onde muda o WE, pois o A20 entra no lugar do WE nessa eeprom.
![MBM29LV800BA MSP55LV160A](https://user-images.githubusercontent.com/120527829/208274538-5c332421-98ab-43c8-bf7a-8f08aaa7371f.PNG)
